<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRES</class>
 <widget class="QMainWindow" name="CRES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="QWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="basicSettings">
       <attribute name="title">
        <string>Basic Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item row="2" column="2" rowspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Example&lt;/span&gt; for GV-SOIL-NPV mixture:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;GV index&lt;/span&gt; = rmse weight * rmse&lt;/p&gt;&lt;p&gt;+ |GV calculated - estimated fraction| * GV weight&lt;/p&gt;&lt;p&gt;+ |SOIL calculated - estimated fraction| * 1&lt;/p&gt;&lt;p&gt;+ |NPV calculated - estimated fraction| * 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" rowspan="4">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>170</width>
             <height>165</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="fractionsBox">
              <property name="title">
               <string>Target fractions for INDEX</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout"/>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="weightsBox">
              <property name="title">
               <string>Weights for INDEX</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_4"/>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="imageGroup">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Image Spectrum (saved as spectral library)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QgsFileWidget" name="browseImage" native="true">
             <property name="toolTip">
              <string>Browse for the spectral library</string>
             </property>
             <property name="filter" stdset="0">
              <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Spectrum:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="spectrumDropDown"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="libraryGroup">
          <property name="title">
           <string>Spectral Library</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QgsFileWidget" name="browseLibrary" native="true">
             <property name="toolTip">
              <string>Browse for the spectral library</string>
             </property>
             <property name="filter" stdset="0">
              <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="2">
              <widget class="QLabel" name="nModelsLabel">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Number of models:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Metadata field with classes:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="classDropDown"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Max. RMSE for SMA fractions:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="maxRMSESpinBox">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>0.025000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advancedSettings">
       <attribute name="title">
        <string>Advanced Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="spacing">
         <number>9</number>
        </property>
        <item row="4" column="0" colspan="3">
         <widget class="QGroupBox" name="shadeCheck">
          <property name="toolTip">
           <string>Select a single shade spectrum</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {border: none}</string>
          </property>
          <property name="title">
           <string>Non-photometric shade: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="topMargin">
            <number>25</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>9</number>
           </property>
           <item row="3" column="0">
            <widget class="QLabel" name="shadeScaleLabel">
             <property name="text">
              <string>Reflectance scale factor for shade signal: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <widget class="QgsSpinBox" name="shadeScaleValue">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QgsFileWidget" name="browseShade" native="true">
             <property name="toolTip">
              <string>Select a single shade spectrum</string>
             </property>
             <property name="filter" stdset="0">
              <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Image Spectrum - Reflectance scale factor:     </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QgsSpinBox" name="libraryScaleValue">
          <property name="toolTip">
           <string>The reflectance scale factor is automatically detected. It is the number by which the array must be divided to get reflectance values between 0 and 1. The user can change this value here.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QgsSpinBox" name="imageScaleValue">
          <property name="toolTip">
           <string>The reflectance scale factor is automatically detected. It is the number by which the array must be divided to get reflectance values between 0 and 1. The user can change this value here.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Spectral library - Reflectance scale factor:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>515</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0" colspan="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_log">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTextBrowser" name="logBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="buttonBar">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="6" rowspan="2">
       <widget class="QGroupBox" name="filterMinFracCheck">
        <property name="title">
         <string>Min. Fraction</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QDoubleSpinBox" name="filterMinFracValue">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-0.500000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>-0.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" rowspan="2">
       <widget class="QGroupBox" name="filterRMSECheck">
        <property name="title">
         <string>RMSE</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QDoubleSpinBox" name="filterRMSEValue">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>0.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.025000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="QGroupBox" name="filterMaxFracCheck">
        <property name="title">
         <string>Max. Fraction</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="filterMaxFracValue">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>1.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>1.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Filter Table&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="nModelsShown">
        <property name="text">
         <string># models shown: 0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>Export selection as new Library</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="nModelsSelected">
        <property name="text">
         <string># models selected: 0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="calculateIndexButton">
        <property name="text">
         <string>Calculate Index</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="calculateFractionsButton">
        <property name="text">
         <string>Calculate SMA fractions</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="tableArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>997</width>
         <height>148</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
